src/meta-srv/src/selector/round_robin.rs

```diff
@@ -125,6 +127,35 @@ impl Selector for RoundRobinSelector {
     }
 }
 
+pub struct FlowPeerAllocator {
+    ctx: SelectorContext,
+    selector: SelectorRef,
+}
+
+impl FlowPeerAllocator {
+    pub fn new(ctx: SelectorContext, selector: SelectorRef) -> Self {
+        Self { ctx, selector }
+    }
+}
+
+#[async_trait::async_trait]
+impl PartitionPeerAllocator for FlowPeerAllocator {
+    async fn alloc(&self, partitions: usize) -> common_meta::error::Result<Vec<Peer>> {
+        self.selector
+            .select(
+                0, // TODO(discord9): cluster id set to 0 for now
```

Comment:
We should add a cluster id parameter to the `alloc` method, which we can set to 0 outside the method.